### ISSUE TITLE
unknown option bar_action_expand and colors not working

### ISSUE THREAD

**Issue by ilnerdchuck** [conformal/spectrwm#401]
I get an error while trying to add `bar_action_expand=1` to set colors for the bar in the action script:
```
bar_color[1]            = rgb:1d/20/21, rgb:fe/80/19, rgb:8e/c0/7c, rgb:d3/86/9b, rgb:b8/bb/26, rgb:a>
bar_color_selected[1]   = rgb:8e/c0/7c
bar_delay               = 5
bar_font_color[1]       = rgb:bb/c5/ff, rgb:e1/ac/ff, rgb:dd/ff/a7, rgb:ff/8b/92, rgb:ff/e5/85, rgb:8>
bar_font_color_selected = black
bar_font                = Hack Nerd Font:size=9:antialias=true
bar_justify             = center
bar_format              = +@bg=2 +A;

Script output:
echo "  $(cpu) |  $(mem) |  $(hdd) | 墳 $(vol) |"
```
 the outbut that i get is "+@bg=2  $(cpu) |  $(mem) |  $(hdd) | 墳 $(vol) |" in the bar

**Comment by LordReg** [comment]
Also,  `bar_action_expand` was implemented in spectrwm 3.3.0, which version are you running?

**Comment by ilnerdchuck** [comment]
Didn't saw that from the popos default repo it pulls the version 3.2, upgraded and it works perfectly thanks
Ps I forgot to port the right code I was trying to move the semicolons as if that was the problem
